targets/DeployDependencies.proj

```diff
@@ -129,6 +129,12 @@
           DestinationFolder="@(RuntimeProjectJson->'%(TestDeploymentDir)')"
           SkipUnchangedFiles="true"
           />
+
+    <Copy SourceFiles="@(ResolvedRuntimeFiles)"
```

Comment:
I'm not sure.  When running tests from Visual Studio, we need runtime dependencies.  At some point, we'll be re-doing our projects and the output paths.